dozer-types/src/models/connection.rs

```diff
@@ -208,13 +208,41 @@ fn get_sslmode(mode: String) -> Result<SslMode, DeserializationError> {
     }
 }
 
-#[derive(Debug, Serialize, Deserialize, JsonSchema, Eq, PartialEq, Clone, Hash)]
+#[derive(Debug, Serialize, Deserialize, JsonSchema, Eq, PartialEq, Clone, Hash, Default)]
 pub struct AerospikeConnection {
     pub hosts: String,
     pub namespace: String,
     pub sets: Vec<String>,
     #[serde(default)]
     pub batching: bool,
+    #[serde(default)]
+    pub replication: ReplicationSettings,
+    pub schemas: Option<ConfigSchemas>,
```

Comment:
`schema` should also be `#[serde(default)]` I think. Otherwise you have to specify something, even if you only use aerospike as sink